docs/docs/tutorial/part-1/index.mdx

```diff
@@ -254,47 +271,37 @@ To connect your code on GitHub to your Gatsby Cloud account, do the following:
 
    ![An empty Gatsby Cloud dashboard](./01-create-a-site-button.png)
 
-1. The next few steps will help you add your site to Gatsby Cloud. First, select **"Import from a Git repository"** and click the **"Next"** button at the bottom of the page.
-
-   ![The "Add a site" screen. The option for "Import from a Git repository" is selected.](./02-import-a-git-repo.png)
+1. The next few steps will help you add your site to Gatsby Cloud. First, in the **"Import from a Git repository"** card click the **"GitHub"** icon to select GitHub as your Git provider.
 
-1. Select **GitHub** as the Git provider.
+   ![The "Add a site" screen. Select the option for "Import from a Git repository".](./02-import-a-git-repo.png)
 
-   ![The "Git provider" tab of the "Add a site" screen. It lists two options: GitHub or GitLab.](./03-select-a-git-provider.png)
-
-1. If this is your first time connecting GitHub to Gatsby Cloud, you'll need to give Gatsby Cloud permission to access your GitHub account. When asked, **"Where should we host this new project?"** click the **"Select destination"** button.
+1. If this is your first time connecting GitHub to Gatsby Cloud, you'll need to give Gatsby Cloud permission to access your GitHub account.
 
   <Announcement style={{marginBottom: "1.5rem"}}>
 
-  **Note:** If you are part of more than one GitHub organization, you will need to first select the organization with which the repository resides at this step before selecting the repository itself.
+    **Note:** If you are part of more than one GitHub organization, you will need to first select the organization with which the repository resides at this step before selecting the repository itself.
```

Comment:
FWIW, this indentation looks suspect to me – could also be related to the Announcement component being using inside the parent `ol`

Reply:
With the latest commit on this branch by Rodney, I'm seeing everything work as expected. He is my personal hero today ❤️